rosdep/python.yaml

```diff
@@ -9323,6 +9323,19 @@ python3-toml:
   gentoo: [dev-python/toml]
   nixos: [python3Packages.toml]
   ubuntu: [python3-toml]
+python3-toppra:
```

Comment:
Please since this is a pip only key please change its name to `python3-toppra-pip`

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1

Reply:
thanks for the review, this has been updated